app/browser/windows.js

```diff
@@ -55,9 +55,11 @@ const getWindowValue = (windowId) => {
 }
 
 const updateWindow = (windowId) => {
-  let windowValue = getWindowValue(windowId)
-  if (windowValue) {
+  const win = BrowserWindow.fromId(windowId)
+  if (win) {
+    const windowValue = getWindowValue(windowId)
     appActions.windowUpdated(windowValue)
+    updateComputedStyles(win)
```

Comment:
The only problem with calling this function from this file is that we don't have document, so you can't do `document.querySelector`. Any idea how to fix this? @bridiver